src/main/java/it/gov/pagopa/arc/connector/bizevents/paidnotice/BizEventsPaidNoticeConnector.java

```diff
@@ -0,0 +1,7 @@
+package it.gov.pagopa.arc.connector.bizevents.paidnotice;
+
+import feign.Response;
+
+public interface BizEventsPaidNoticeConnector {
+    Response getPaidNoticeList(String fiscalCode, String continuationToken, Integer size, Boolean isPayer, Boolean isDebtor, String orderBy, String ordering);
```

Comment:
Return a Response object will expose to the service layer the source of the data, which should be hidden by this layer!
It is recommended to return a DTO